JitsiMeetJS.js

```diff
@@ -57,6 +61,7 @@ var LibJitsiMeet = {
     },
     logLevels: Logger.levels,
     mediaDevices: JitsiMediaDevices,
+    environment: RTCBrowserType,
```

Comment:
I need to construct different message depending on browser type for overlay with guidance for allowing camera and microphone access.
We can re-implement browser detection in jitsi-meet if that's OK.